dss_datamover/logger.py

```diff
@@ -163,10 +162,15 @@ def logging(self, queue, lock, stop_logging):
                         (message_level, message_value) = message
                         print("{}: {}".format(LOGGING_LEVEL[message_level], message_value))
                         fh.write(str(time.ctime()) + ": " + LOGGING_LEVEL[message_level] + ": " + message_value + "\n")
+                    elif message is None:
+                        stop_flag = True
+                        break
                     else:
                         fh.write(str(time.ctime()) + ": " + message + "\n")
 
-                if stop_logging.value and queue.qsize() == 0:
```

Comment:
We should also keep this condition. If stop_logging is set and queue is empty then we should also print a message and break here.

Reply:
We are experiencing issues with MP queues. Even though there are no elements in queues, the queue calls are getting hung. Thats why we are doing a workaround.